R/theme_cmap.R

```diff
@@ -19,10 +19,14 @@
 #'@examples
 #'
 #'\dontrun{
+#'
+#' # The only way to place the origin line (`hline`, `vline`) behind any data geoms

```

Comment:
tweaked this example. 

This example reveals an imperfection we haven't managed yet -- looks like CMAP graphics try to have an axis line that is also a data line. (see [here](https://github.com/CMAP-REPOS/cmapplot/wiki/Example-Outputs)). This is a gap we should still fill if possible?